### PR TITLE
improve readme and docrs

### PR DESCRIPTION
**Pull request by gwbres**

### CODE REVIEW

File: rinex/README.md [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
 [![rustc](https://img.shields.io/badge/rustc-1.61%2B-blue.svg)](https://img.shields.io/badge/rustc-1.61%2B-blue.svg)
-
-[![Rust](https://github.com/gwbres/rinex/actions/workflows/rust.yml/badge.svg)](https://github.com/gwbres/rinex/actions/workflows/rust.yml)
 [![crates.io](https://docs.rs/rinex/badge.svg)](https://docs.rs/rinex/badge.svg)
-[![crates.io](https://img.shields.io/crates/d/rinex.svg)](https://crates.io/crates/rinex) 
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue?style=flat-square)](https://github.com/gwbres/rinex/blob/main/LICENSE-APACHE)
-[![License](https://img.shields.io/badge/license-MIT-blue?style=flat-square)](https://github.com/gwbres/rinex/blob/main/LICENSE-MIT)
 
-## Parser
+*RINEX* is a *Georust* ecosystem that aims at supporting
```
Comment (lnicola) [inline rinex/README.md:7]:
```suggestion
*RINEX* is a crate in the *GeoRust* ecosystem that aims at supporting
```

File: rinex/README.md [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
+most common RINEX formats, for both data analysis and data production,
+without performance compromises.
+
+It aims at providing a modern and credible alternatives to tools like `teqc`
```
Comment (lnicola) [inline rinex/README.md:11]:
```suggestion
It aims at providing a modern and credible alternatives to tools like `teqc`.
```

File: rinex/src/lib.rs [hunk 3]
```diff
@@ -1,7 +1,5 @@
-//! This library provides a set of tools to parse, analyze
-//! and process RINEX files.
-//! Refer to README and documentation hosted here
-//! <https://github.com/georust/rinex>
+#![doc(html_logo_url = "https://raw.githubusercontent.com/georust/meta/master/logo/logo.png")]
```
Comment (lnicola) [inline rinex/src/lib.rs:1]: Is this what people do these days? I don't think raw.githubusercontent.com was supposed to be used as a CDN, and e.g. it doesn't set the `Content-Type` on the response.

File: rinex/README.md [hunk 4]
```diff
@@ -1,29 +1,40 @@
...
-## Production (writer)
+The `sbas` feature enables one method to select appropriate augmentation system
+based on current location on Earth.
 
-Once a RINEX structure is parsed, it is possible to dump it into a file,
-and possibily rework it in the meantime.
+## License
 
-## File naming convention
+Licensed under either
```
Comment (lnicola) [inline rinex/README.md:38]:
```suggestion
Licensed under either of:
```

File: rinex/src/lib.rs (hunk 3 above)
Comment (gwbres) [reply rinex/src/lib.rs:1]: that's what georust::geo does at least ?

Comment (lnicola) [reply rinex/src/lib.rs:1]: I don't think it should, but fair enough :-).

**Review by lnicola — APPROVED** [review]

**Pull request merged**